recirq/benchmarks/rep_rate/rep_rate_calculator.py

```diff
@@ -0,0 +1,263 @@
+"""
+  Class to calculate latency and repetition rate.
+
+  This class should replicate the acquisition of date for
+  rep rate and latency for processor data sheets.
+"""
+import random
+import cirq
+import cirq.google as cg
+import time
+import sympy
+
+from typing import List, Optional, Tuple
+
+
+class RepRateCalculator:
+    """Class to perform simple timing experiments using hardware.
+
+    This class calculates the repetition rate for various
+    types of circuits.  This is the approximate type per repetition
+    (or shot) to execute this on the device.  This rate varies widely
+    based on width, depth, and amount of sweeps/batching in the
+    circuit.  This class will help test various rates on the device.
+
+    This class is used to generate statistics on the processor datasheets
+    for latency and repetition rates.
+    """
+
+    def __init__(self,
+                 project_id: str,
```

Comment:
consider making this optional and using `cg.get_engine` which will use the google cloud environment variable to get the project id if one is not specified

Reply:
Restructured.  Let me know what you think.